web/src/scripts/webflow/licenseSync.mjs

```diff
@@ -240,10 +274,12 @@ if (process.env.NODE_ENV === "test") {
   })();
 } else if (argsInclude("--preview")) {
   await (async () => {
+    console.info("---- To be deleted: -----");
+    console.info((await getUnUsedLicenses()).map((it) => it.fieldData.name));
     console.info("---- To be created: -----");
-    console.info((await getNewLicenses()).map((it) => it.filename));
+    console.info((await getNewLicenses()).map((it) => it.name));
     console.info("---- To be updated: -----");
-    console.info((await getLicensesAlreadyInWebflow()).map((it) => it.filename));
+    console.info((await getLicensesAlreadyInWebflow()).map((it) => it.name));
```

Comment:
I think these fields are more useful as they facilitate actual verification of existence on the web-flow site. IE you can search for these.